Header header
int32 application
time timestamp
time startup
string terminal
float64 can_str
float64 can_vel
float64 can_gas
float64 can_brake
float64 can_seatbelt
float64 can_water_temperature
float64 acc_x
float64 acc_y
float64 acc_z
float64 lat
float64 lon
float64 roll
float64 pitch
float64 yaw
string img
float64 cal_orientation_x
float64 cal_orientation_y
float64 cal_orientation_z
float64 gravity_x
float64 gravity_y
float64 gravity_z
float64 gyroscope_x
float64 gyroscope_y
float64 gyroscope_z
float64 light
float64 linear_acc_x
float64 linear_acc_y
float64 linear_acc_z
float64 magnetic_field_x
float64 magnetic_field_y
float64 magnetic_field_z
float64 orientation_x
float64 orientation_y
float64 orientation_z
float64 pressure
float64 proximity
float64 rotation_vector_x
float64 rotation_vector_y
float64 rotation_vector_z
float64 rotation_vector_c
float64 temperature
float64 ambient_temperature
float64 relative_humidity
float64 bearing
float64 altitude
float64 accuracy
float64 speed
float64 env_battery
float64 env_temperature
